scripts/integration-test-notebook.bat

```diff
@@ -0,0 +1,22 @@
+:: Runs unit tests for Extensions
+
+setlocal
+
+pushd %~dp0\..
```

Comment:
Will this be included in some nightly runs? @alanrenmsft  or @chlafreniere  can you advise on how we should use these?

Reply:
Removed this file.